packages/connect/src/protocol-connect/handler-factory.ts

```diff
@@ -78,11 +85,16 @@ import {
 import { codeToHttpStatus } from "./http-status.js";
 import { errorToJsonBytes } from "./error-json.js";
 import { trailerMux } from "./trailer-mux.js";
-import { requireProtocolVersion } from "./version.js";
+import {
+  requireProtocolVersionHeader,
+  requireProtocolVersionParam,
+} from "./version.js";
 import { parseTimeout } from "./parse-timeout.js";
+import { paramMessage, paramBase64 } from "./query-params.js";
```

Comment:
I ran organize imports. I didn't really realize it handled our import style correctly, because some other tsserver functions don't seem to handle it very well. Thanks for the tip.